test/runtime/utils/profileHelper.test.ts

```diff
@@ -0,0 +1,86 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import * as fs from "fs";
+import * as os from "os";
+import * as jsonfile from "jsonfile";
+
+import { listExistingProfileNames, createConfigFileIfNotExists } from "../../../src/runtime/lib/utils/profileHelper";
+import * as jsonUtility from "../../../src/runtime/lib/utils/jsonUtility";
+
+describe("ProfileHelper tests", () => {
+    let sandbox: sinon.SinonSandbox;
+    beforeEach(() => {
+        sandbox = sinon.createSandbox();
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    describe("Function listExistingProfileNames tests", () => {
+        it("Should return null when askConfig not exist", () => {
+            sandbox.stub(fs, "existsSync").returns(false);
```

Comment:
One question about the code style, should the test files keep the same style or not? According to this https://mochajs.org/#arrow-functions, I probably will not use arrow functions.